ftplugin/php/splitjoin.vim

```diff
@@ -1,6 +1,6 @@
 let b:splitjoin_split_callbacks = [
       \ 'sj#js#SplitArray',
-      \ 'sj#php#SplitArray',
+      \ 'sj#php#SplitBraces',
```

Comment:
Mmmm this is pretty odd, I can't reproduce that behavior. Where do you put the cursor? I'm putting it right on opening curly brace and I don't experience that bug.